### ISSUE TITLE
Write zstd compressed ROOT files

### ISSUE THREAD

**Issue by reikdas** [scikit-hep/uproot#470]

**Comment by jpivarski** [comment]
Check on this: it might already be there. @oshadura contributed a high-quality PR adding zstd support to Uproot—I can't remember, but I think it might have _written_ zstd data as well.

**Comment by reikdas** [comment]
It was just read - https://github.com/scikit-hep/uproot/pulls?q=type%3Apr+author%3Aoshadura

**Comment by oshadura** [comment]
Yup, sorry, we were talking to add writing in zstd after official release :)
